themes/classic/_dev/js/checkout.js

```diff
@@ -14,17 +14,16 @@ function setupMyCheckoutScripts() {
 
   $('.js-terms a').on('click', (event) => {
     event.preventDefault();
+    var url = $(event.target).attr('href');
+    if (url) {
+      url += `?content_only=1`;
```

Comment:
:+1:  can you please put the comment ? the feature is still broken in on pretty-url mode ...